packages/@aws-cdk/aws-ec2-alpha/README.md

```diff
@@ -231,6 +231,63 @@ new Route(this, 'DynamoDBRoute', {
 });
 ```
 
+## VPC Peering Connection
+
+VPC peering connection allows you to connect two VPCs and route traffic between them using private IP addresses. The VpcV2 construct supports creating VPC peering connections through the `VPCPeeringConnection` construct from the `route` module.
+
+This is an example of how to create a VPC peering connection between two VPCs. The process involves 3 main steps:
+
+**Step 1: Acceptor Account Stack**
+
+Create a restrictive IAM role in the acceptor VPC account. This role will be used to grant limited permissions for accepting the peering request from the requestor account.
+
+```ts
+// Step 1: Acceptor Account Stack
+const stack = new Stack();
+
+const acceptorVpc = new VpcV2(this, 'VpcA', {
+  primaryAddressBlock: IpAddresses.ipv4('10.0.0.0/16'),
+});
+
+const acceptorRoleArn = acceptorVpc.createAcceptorVpcRole('987654321098')
+```
+
+**Step 2: Requestor Account Stack**
+
+Initiate the peering connection request from the requestor VPC.
+
+**Step 3: Route Table Configuration**
+
+Update route tables in both VPCs to enable traffic flow. If a route is added to the requestor stack, information will be able to flow from the requestor VPC to the acceptor VPC, but not in the reverse direction. For bi-directional communication, routes need to be added in both VPCs from their respective stacks.
+
```

Comment:
from my understanding there is also some step to be added from Console or by CLI to add the route to the VPC route table. Can you add this step here, also will be good if you add the CLI commands.

Reply:
for bi-directional communication, please add a separate section at the end for it, and explain the steps, and add the CLI commands to do these steps.

Reply:
one more thing, shall you check how can we delete this connection, and what are the required steps.

Reply:
added in readme to address comments! [b926d86](https://github.com/aws/aws-cdk/pull/31645/commits/b926d864a3746cef4aaeee2aa55e9ed84c434842)